### PR TITLE
Implemented Cassandra Worker

### PR DESCRIPTION
**Pull request by vajda**

### CODE REVIEW

File: berserker-cassandra/src/main/java/io/smartcat/berserker/cassandra/worker/CassandraWorker.java [hunk 1]
```diff
@@ -0,0 +1,133 @@
...
+    public CassandraWorker(List<InetSocketAddress> connectionPointsWithPorts, String keyspace, boolean async,
+            List<String> bootstrapDDLCommands, List<PreparedStatement> prepStatements) {
+        if (connectionPointsWithPorts == null || connectionPointsWithPorts.isEmpty()) {
+            throw new IllegalArgumentException("List of connection points with ports cannot be null nor empty");
+        }
+        if (keyspace == null || keyspace.isEmpty()) {
+            throw new IllegalArgumentException("Keyspace cannot be null nor empty.");
+        }
+        cluster = Cluster.builder().addContactPointsWithPorts(connectionPointsWithPorts).build();
+        if (bootstrapDDLCommands == null || bootstrapDDLCommands.isEmpty()) {
```
Comment (mgobec) [inline berserker-cassandra/src/main/java/io/smartcat/berserker/cassandra/worker/CassandraWorker.java]: Simplify this to:
session = cluster.connect();
if (bootstrap)
    for (command)
        execute
session.execute("USE " + keyspace + ";")

File: berserker-cassandra/src/main/java/io/smartcat/berserker/cassandra/worker/CassandraWorker.java [hunk 2]
```diff
@@ -0,0 +1,133 @@
...
+            for (String command : bootstrapDDLCommands) {
+                session.execute(command);
+            }
+            session.execute("USE " + keyspace + ";");
+        }
+        preparedStatements = new HashMap<>();
+        if (prepStatements != null && !prepStatements.isEmpty()) {
+            for (PreparedStatement prepStatement : prepStatements) {
+                RegularStatement toPrepare = new SimpleStatement(prepStatement.getQuery());
+                com.datastax.driver.core.PreparedStatement prepared = session.prepare(toPrepare);
```
Comment (mgobec) [inline berserker-cassandra/src/main/java/io/smartcat/berserker/cassandra/worker/CassandraWorker.java]: You can remove intermediate variable so there is no need for the explicit package name
perparedStatements.put(id, session.prepare(toPrepare))

File: berserker-cassandra/src/main/java/io/smartcat/berserker/cassandra/worker/CassandraWorker.java [hunk 3]
```diff
@@ -0,0 +1,133 @@
...
+        if (async) {
+            session.executeAsync(toExecute);
+        } else {
+            session.execute(toExecute);
+        }
+    }
+
+    private ConsistencyLevel getConsistencyLevel(Map<String, Object> queryMetadata) {
+        String consistency = (String) queryMetadata.get(CONSISTENCY_LEVEL);
+        if (consistency == null || consistency.isEmpty()) {
```
Comment (mgobec) [inline berserker-cassandra/src/main/java/io/smartcat/berserker/cassandra/worker/CassandraWorker.java]: Can you use default consistency level of ONE if its not specified or you want to enforce all configuration options to be specified?

File: berserker-cassandra/src/main/java/io/smartcat/berserker/cassandra/worker/CassandraWorker.java (hunk 1 above)
Comment (vajda) [inline berserker-cassandra/src/main/java/io/smartcat/berserker/cassandra/worker/CassandraWorker.java]: I agree.

File: berserker-cassandra/src/main/java/io/smartcat/berserker/cassandra/worker/CassandraWorker.java (hunk 2 above)
Comment (vajda) [inline berserker-cassandra/src/main/java/io/smartcat/berserker/cassandra/worker/CassandraWorker.java]: I agree

File: berserker-cassandra/src/main/java/io/smartcat/berserker/cassandra/worker/CassandraWorker.java (hunk 3 above)
Comment (vajda) [inline berserker-cassandra/src/main/java/io/smartcat/berserker/cassandra/worker/CassandraWorker.java]: Idea was to have it per request, not global. Having it optional with fallback to ONE is still OK, and probably a bit more user friendly. I will make it optional.

**Pull request merged**